### PR TITLE
Update Buildozer osx target to work with latest kivy 2.2.0

### PR DESCRIPTION
**Pull request by akshayaurora**
`buildozer osx debug` should work now.

### CODE REVIEW

File: buildozer/targets/osx.py [hunk 1]
```diff
@@ -169,7 +163,7 @@ def get_custom_commands(self):
         return result
 
     def get_available_packages(self):
-        return ['kivy']
+        return ['kivy', 'python3']
```
Comment (AndiEcker) [inline buildozer/targets/osx.py:166]: pyver = self.buildozer.config.get('app', 'osx.python_version')
        return ['kivy', f'python{pyver}']

**Review by misl6 — COMMENTED** [review]
Hi @akshayaurora !

I've left some comments, let me know your thoughts 😀

File: buildozer/targets/osx.py [hunk 2]
```diff
@@ -97,6 +90,7 @@ def build_package(self):
         bc = self.buildozer.config
         bcg = bc.get
         package_name = bcg('app', 'package.name')
+        pyver = bcg('app', 'osx.python_version')
```
Comment (misl6) [inline buildozer/targets/osx.py:93]: A PEP8 test is now failing here :)

**Review by misl6 — CHANGES_REQUESTED** [review]

**Review by misl6 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**